index.html

```diff
@@ -1,15 +1,141 @@
 <!DOCTYPE html>
-<html >
+<html lang="en-US">
 	<head>
 		<meta charset="UTF-8">
 		<meta name="viewport" content="width=device-width, initial-scale=1.0">
-		<title></title>
-		<style >
-			
-		</style>
+		<title>Intuit. Signup</title>
 		<link rel="stylesheet" href="stylesheet.css" />
 	</head>
 	<body>
-		<h1>Intuit Login Page clone</h1>
+		<section>
```

Comment:
You seems to be overusing the `section` tag, note that the `section` tag should **mostly** be used only for a sectional part of the page that has its own heading, that way your section element, will have it's own `heading` element.

See this errors/warning for more details https://validator.w3.org/nu/?doc=https%3A%2F%2Frawcdn.githack.com%2Fadejam%2Fintuit-signup-page-clone%2F3588299e896dbe4658a418577085bc0de193a739%2Findex.html

Reply:
Thanks for the review. I'll remove all sections without its own headings.